packages/medusa-payment-stripe/src/core/stripe-base.ts

```diff
@@ -19,7 +19,7 @@ abstract class StripeBase extends AbstractPaymentProcessor {
   static identifier = ""
 
   protected readonly options_: StripeOptions
-  protected stripe_: Stripe
+  stripe_: Stripe
```

Comment:
Todo: should stay protected and create a getter get stripe that return this.stripe_, we don't want a setter on this